paparazzi/src/main/java/app/cash/paparazzi/deprecated/com/android/ide/common/resources/deprecated/MultiResourceFile.java

```diff
@@ -19,19 +19,24 @@
 import app.cash.paparazzi.deprecated.com.android.ide.common.resources.deprecated.ValueResourceParser.IValueResourceRepository;
 import app.cash.paparazzi.deprecated.com.android.io.IAbstractFile;
 import app.cash.paparazzi.deprecated.com.android.io.StreamException;
+import kotlin.text.Charsets;
```

Comment:
This is a pure Java file, I would keep it that way:
```suggestion
import java.nio.charset.StandardCharsets;
```